Documentation/compatibility/certificate-eku-oid-validation.md

```diff
@@ -0,0 +1,48 @@
+## Certificate EKU OID validation
+
+### Scope
+Minor
+
+### Version Introduced
+4.6
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+
+Starting with .NET Framework 4.6, the <xref:System.Net.Security.SslStream> or <xref:System.Net.ServicePointManager> classes perform enhanced key use (EKU) object identifier (OID) validation. An enhanced key usage (EKU) extension is a collection of object identifiers (OIDs) that indicate the applications that use the key. EKU OID validation uses remote certificate callbacks to ensure that the remote certificate has the correct OIDs for the intended purpose.  
+
+- [X] Quirked 
+- [ ] Build-time break
+
+### Recommended Action
+
+If this change is undesirable, you can disable certificate EKU OID validation by adding the following switch to the [`\<AppContextSwitchOverrides>` element](https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element) in the [`\<runtime> section](https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/runtime-element) of your app configuration file:
+
+```xml
+<runtime>
+   <AppContextSwitchOverrides   
+          value="Switch.System.Net.DontCheckCertificateEKUs=true" /> 
+</runtime>
+```
+> [!IMPORTANT] 
+> This setting is provided for backward compatibility only. Its use is otherwise not recommended.
+
+ 
+### Affected APIs
+* `T:System.Net.Security.SslStream`
+* `T:System.Net.ServicePointManager`
+* `T:System.Net.Http.HttpClient`
+* `T:System.Net.Mail.SmtpClient`
+* `T:System.Net.HttpWebRequest`
+* `T:System.Net.FtpWebRequest`
+
+### Category
+Networking
+
+<!--
+    ### Original Bug #364538 
```

Comment:
Do customers see this text about the bug numbers? That's ok since we are now listing bug numbers in release notes for which bugs are fixed etc.

Reply:
@davidsh, they see them on GitHub only if they view the raw source; they're suppressed from the ordinary view. Since they're comments, they're not published on docs.